packages/epetra/src/Epetra_MultiVector.h

```diff
@@ -985,6 +985,7 @@ class EPETRA_LIB_DLL_EXPORT Epetra_MultiVector: public Epetra_DistObject, public
   //@}
 
   // Expert-only function
+#undef Reduce
```

Comment:
Can you `#undef` something that's not defined?  Perhaps we should do this:
```c++
#ifdef Reduce
#undef Reduce
#endif 
```

Reply:
yeah that's safer, I just copy/pasted in rush, thanks!

Reply:
Will update in a moment

Reply:
@mhoemmen updated, thanks!